manage_breast_screening/config/system_test_setup.py

```diff
@@ -26,3 +26,12 @@ def setUp(self):
 
     def tearDown(self):
         self.page.close()
+
+    def expect_validation_error(self, id: str, fieldset_legend: str, error_text: str):
+        summary_box = self.page.locator(".nhsuk-error-summary")
+        error_link = summary_box.locator(f"a[href='#{id}']")
+        expect(error_link).to_have_text(error_text)
+
+        fieldset = self.page.locator('fieldset').filter(has_text=fieldset_legend)
+        error_span = fieldset.locator("span").filter(has_text=error_text)
+        expect(error_span).to_have_id(id)
```

Comment:
Once we fix the focus, then I think it would be better to call `click` here rather than assert against the ID. Happy to leave it as it is for now though